fabtests/multinode/src/core.c

```diff
@@ -244,6 +247,10 @@ int multi_msg_send()
 
 		offset = state.sends_posted % opts.window_size;
 		assert(tx_ctx_arr[offset].state == OP_DONE);
+		if (ft_check_opts(FT_OPT_PERF)) {
+			timer_index = (state.iter * pm_job.num_ranks) + state.cur_target;
```

Comment:
The timer index calculation is repeated several times.  Please create a static inline helper function to return this.  E.g.
```
static inline int timer_index(int iter, int dest_rank)
{
    return iter * pm_job.num_ranks + dest_rank;
}
```
assuming I interpreted cur_target correctly.